lib/v3/script.js

```diff
@@ -240,79 +245,61 @@ class ScriptParser extends EventEmitter {
                 const specifiers = node.specifiers;
 
                 if (declaration) {
-                    const exportNodeComment = getCommentFromSourceCode(node, parseContext.sourceCode, { defaultVisibility: 'public', useLeading: true, useTrailing: false });
+                    const exportNodeComment = getCommentFromSourceCode(
+                        node,
+                        parseContext.sourceCode,
+                        { defaultVisibility: 'public', useLeading: true, useTrailing: false }
+                    );
 
                     if (declaration.type === 'VariableDeclaration') {
-                        const variables = parseVariableDeclaration(declaration);
-
-                        variables.forEach(variable => {
-                            this.emit('data', variable, parseContext, 'public', exportNodeComment);
-
-                            if (variable.declarator.init) {
-                                const initNode = variable.declarator.init;
-
-                                if (initNode.type === 'CallExpression') {
-                                    const callee = initNode.callee;
-
-                                    if (initNode.arguments) {
-                                        this.parseBodyRecursively(initNode.arguments, parseContext, level + 1);
-                                    }
-
-                                    if (callee.type === 'Identifier' && this.dispatcherConstructorNames.includes(callee.name)) {
-                                        this.dispatcherNames.push(variable.name);
-                                    }
-                                } else if (initNode.type === 'ArrowFunctionExpression') {
-                                    if (initNode.body) {
-                                        this.parseBodyRecursively(initNode.body, parseContext, level + 1);
-                                    }
-                                }
-                            }
-                        });
-
-                        return;
```

Comment:
This is the code path for exported nodes, but as you can see, the addition of identifiers is missing.